utils/kata-manager.sh

```diff
@@ -219,7 +219,24 @@ github_get_release_file_url()
 
 	case "$url" in
 		*kata*)
-			regex="kata-static-${version}-${arch_regex}.tar.zst" ;;
+			local suffix="xz"
+			local last_xz_target="3.20.0"
+			IFS='.' read -r -a version_parts <<< "${version}"
+			IFS='.' read -r -a last_xz_target_parts <<< "${last_xz_target}"
+
+			for i in 0 1 2; do
+				local v=${version_parts[i]:-0}
+				local t=${last_xz_target_parts[i]:-0}
+
+				if (( v < t )); then
+					break
+				elif (( v > t )); then
+					suffix="zst"
+					break
+				fi
+			done
+			
+			regex="kata-static-${version}-${arch_regex}.tar.${suffix}" ;;
```

Comment:
Since this is a regex, couldn't it just be


```suggestion
			regex="kata-static-${version}-${arch_regex}.tar.(xz|zst)" ;;
```

Reply:
Oh man, I just missed this would be evaluated as a regex (which is just in the name of the var).